docs/src/understanding-deis/components.md

```diff
@@ -41,18 +42,22 @@ The router component uses [Nginx][] to route traffic to application containers.
 
 ## Store
 
-The store component uses [Ceph][] to store data for Deis components
-which need to store state, including [Registry][], [Database][].
+The Store component uses [Minio][] to store data for Deis components
+which need to store state, including the [Registry][] and the [Database][].
+
+It is highly recommended that the Store's data is backed by a persistent volume in the event of
```

Comment:
same idea here - if you _do_ use minio, you should back it by a persistent volume